client/packages/common/src/intl/locales/en/patients.json

```diff
@@ -19,6 +19,9 @@
   "label.visit-start": "Start",
   "label.visit-end": "End",
   "label.clinician": "Clinician",
+  "label.additional-info": "Additional Info",
+  "label.visit-notes": "Visit notes, other problems",
```

Comment:
Maybe just "Notes"? mainly got tripped over ", other problems" which was not quite clear to me...